docs/azure_arc_jumpstart/azure_arc_servers/oci/oci_terraform_ubuntu/_index.md

```diff
@@ -0,0 +1,199 @@
+ 
+      ![Screenshot of OCI cloud console](./01.png)
+    
+ * After obtaining User OCID, navigate to [Compartments](https://cloud.oracle.com/identity/compartments). Click on the Tenancy name and find the OCID --then click on show. Copy the Comartment OCID to clipboard and paste it to the editor of choice..   
+ 
+    ![Screenshot of CompartmentID OCI cloud console](./02.png)
+    
+ Tenancy OCID
+* After finding the user OCID, go to [OCI Tenancy](https://cloud.oracle.com/tenancy), navigate to the OCID for Tenancy and click on show. Copy the Tenancy OCID to clipboard and paste it to the editor of choice.
+  
+   ![Screenshot of TenancyID OCI cloud console](./03.png)
+  
+ Create the API SSL Key
+ 
+ * You will need an SSL key pair to enable Terraform to connect to the OCI API under your identity. Start by generating a key
+
+    ```shell
+    $openssl genrsa -out oci_api_key.pem 2048
+    ```
+* set file access to owner only read and write
+
+  ```shell
+   $chmod go-rwx oci_api_key.pem
+  ```
+* generate the public half of the key pair
+ 
+   ```shell
+    $openssl rsa -pubout -in oci_api_key.pem -out oci_api_key_public.pem
+  ```
+
+* Copy the contents of the public key to the clipboard using gvim, pbcopy, xclip or a similar tool (you'll need to paste the value into the Console later).
+ 
+   ```shell
+     $cat oci_api_key_public.pem | gvim &
+  ```
+
+The public key needs to be added to your user account in the OCI console. Open the [account page](https://cloud.oracle.com/identity/users/) for your user, navigate to "API Keys" and select the "Add Public Key" button. Copy and paste the contents of the oci_api_key_public.pem file in the box of the "Add Public Key" dialog as shown below.
+
+  ![Screenshot of OCI cloud console](./04.png)
+
+ ## Configure Terraform
+
+Before executing the Terraform plan, you must export the environment variables which will be used by the plan. These variables are based on your Azure subscription and tenant, the Azure service principal, and the OCI OCIDs and keys you just created.
+
+* Retrieve your Azure Subscription ID and tenant ID using the ```az account list``` command.
+
+* The Terraform plan creates resources in both Microsoft Azure and OCI. It then executes a script on an OCI compute instance to install the Azure Arc agent and all necessary artifacts. This script requires certain information about your OCI and Azure environments. Edit [*scripts/vars.sh*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/oci/ubuntu/terraform/scripts/vars.sh) and update each of the variables with the appropriate values.
+
+    * TF_VAR_tenancy_ocid=Oracle tenanacy OCID
+    * TF_VAR_user_ocid=Oracle user OCID
+    * TF_VAR_fingerprint=Oracle API Key FingerPrint
+    * TF_VAR_private_key_path=oci_api_key.pem
+    * TF_VAR_ssh_public_key=$(cat my_oci_key.pub) 
+    * TF_VAR_region=us-ashburn-1
```

Comment:
Do not pre-populate

Reply:
What are you asking?

Reply:
Add brackets around placeholder values <>

Reply:
added <>